CI builds fail randomly with `assertion failed` in Node

CI builds fail randomly with the following:
```
yarn run v1.22.4
$ mkdirp dist/res && react-native bundle --entry-file index.js --platform android --dev true --bundle-output dist/main.android.jsbundle --assets-dest dist/res --reset-cache
warning: the transform cache was reset.
Loading dependency graph, done.
info Writing bundle output to:, dist/main.android.jsbundle
info Done writing bundle output
info Copying 2 asset files
info Done copying assets
Assertion failed: (!uv__is_closing(handle)), function uv_close, file ../deps/uv/src/unix/core.c, line 117.
/bin/sh: line 1:  2547 Abort trap: 6           react-native bundle --entry-file index.js --platform android --dev true --bundle-output dist/main.android.jsbundle --assets-dest dist/res --reset-cache
error Command failed with exit code 134.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
##[error]Process completed with exit code 134.
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -75,7 +75,7 @@ jobs:
       - name: Build
         run: |
           set -eo pipefail
-          yarn build:ios
+          yarn build:ios || yarn build:ios
           pod install --project-directory=ios
           ../scripts/xcodebuild.sh ios/Example.xcworkspace build-for-testing
         working-directory: example
@@ -114,7 +114,7 @@ jobs:
       - name: Build
         run: |
           set -eo pipefail
-          yarn build:ios
+          yarn build:ios || yarn build:ios
           if [[ ${{ matrix.template }} == ios ]]; then
             pod install
             ../scripts/xcodebuild.sh TemplateExample.xcworkspace build
@@ -160,7 +160,7 @@ jobs:
         shell: bash
         run: |
           set -eo pipefail
-          yarn build:android
+          yarn build:android || yarn build:android
           pushd android 1> /dev/null
           ./gradlew clean build check test
         working-directory: example
@@ -205,7 +205,7 @@ jobs:
       - name: Build
         run: |
           set -eo pipefail
-          yarn build:android
+          yarn build:android || yarn build:android
           [[ -d android ]] && pushd android 1> /dev/null
           ./gradlew clean build check test
         shell: bash
@@ -233,7 +233,7 @@ jobs:
       - name: Build
         run: |
           set -eo pipefail
-          yarn build:macos
+          yarn build:macos || yarn build:macos
           pod install --project-directory=macos
           ../scripts/xcodebuild.sh macos/Example.xcworkspace build-for-testing
         working-directory: example
@@ -272,7 +272,7 @@ jobs:
       - name: Build
         run: |
           set -eo pipefail
-          yarn build:macos
+          yarn build:macos || yarn build:macos
           if [[ ${{ matrix.template }} == macos ]]; then
             pod install
             ../scripts/xcodebuild.sh TemplateExample.xcworkspace build

